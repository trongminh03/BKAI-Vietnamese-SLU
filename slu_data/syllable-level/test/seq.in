đóng hộ cái lò nướng phát
