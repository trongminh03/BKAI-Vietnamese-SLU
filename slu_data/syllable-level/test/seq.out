B-command O O B-device I-device O 
